Monika After Story/game/script-topics.rpy

```diff
@@ -14196,6 +14196,68 @@ label monika_translating_poetry:
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_soft_rains",
+            category=['literature'],
+            prompt="Soft Rains",
+            random=True,
+            aff_range=(mas_aff.LOVE, None)
+        )
+    )
+
+label monika_soft_rains:
+    m 1rkc "..."
+    m 2rkc "..."
+    m 2ekc "Hey [player],{w=0.5} I've been doing a little more reading recently, and I stumbled across something that..."
+    m 2ekc "Well,{w=0.4} gave me something to think about..."
+    m 1eka "I'd really love to share it with you, [player]."
+    m "So,{w=0.5} how about it?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "So, how about it?{fast}"
+        "Sure!":
+            m 1eub "Okay!"
+            m 1dsc "..."
+            m 1dsd "{i}There will come soft rains and the smell of the ground, {w=0.8}and swallows circling with their shimmering sound...{/i}"
+            m 1dsc "{i}And frogs in the pools singing at night, {w=0.8}and wild plum trees in tremulous white...{/i}"
+            m 3dsd "{i}Robins will wear their feathery fire, {w=0.8}whistling their whims on a low fence-wire...{/i}"
+            m 3dsc "{i}And not one will know of the war, {w=0.8}not one, {w=0.8}will care at last when it is done...{/i}"
+            m 6dsd "{i}Not one would mind, neither bird nor tree, {w=0.8}if mankind perished utterly...{/i}"
+            m 6dkc "{i}And Spring herself, when she woke at dawn..."
+            m 6ekd "{i}Would scarcely know that we were gone.{/i}"
+            m 2ekc "..."
+            m 2eka "I really wanted to talk to you about this one, [player]..."
+            m 2eka "The author, {w=0.5}{i}Sara Teasdale{/i},{w=0.5} penned this poem in a world consumed by war."
+            m 2rka "I know things are {i}relatively{/i} peaceful now..."
+            m 2ekc "But...{w=0.3}the message remains, doesn't it?"
+            m 1rksdlc "The world really {i}wouldn't{/i} care if we just...{w=0.3}{i}disappeared{/i}."
+            m 1eksdlc "But it made me think, [player]. If something - {w=0.6}"
+            extend 1rksdld "{i}anything{/i}{w=0.6} - happened to you,{w=0.6} I wouldn't know either..."
+            m 1eksdld "I know the poem was for {i}everyone{/i} being gone, but..."
+            m 1rksdld "I-{w=0.7}I wouldn't..."
+            m 6dkc "..."
+            m 6dktpd "[player],{w=0.7} I..."
+            m 6dktuc "..."
+            m 2dktda "..."
+            m 1ektua "I love you, [player]. {w=0.8}I love you so very much..."
+            m 1ektdc "Please, {w=0.6}always be careful for me, {w=0.8}okay?"
+            m 1ektda "I might not know,{w=0.6} but I will {i}always{/i} care,{w=0.6} [player]."
+            m 2ektda "And...{w=0.6}[player]?"
+            m 1ektda "Thank you."
+            m 6rktda "..."
+            m 6eka "..." 
+            m 2ekbsa "So..."
+            m 1ekbfa "What else shall we do today, my love?"
+            return "love"
```

Comment:
To keep the returns a little cleaner (we've found bugs as a result of this before), we'll do this
```suggestion
            $ masILY()
```

Reply:
I can't test this function call since it doesn't appear to be under the content branch (pulled a few minutes ago to bring up to date); I'm assuming it's already in master, so should be fine?

Reply:
Fixed suggestion

Reply:
Ah, just spotted the correct - bear with me

Reply:
Fixed.